span.go

```diff
@@ -174,10 +184,12 @@ func CanonicalizeBaggageKey(key string) (string, bool) {
 	return strings.ToLower(key), true
 }
 
-// StartChildSpan is a simple helper to start a child span given only its parent (per StartSpanOptions.Parent) and an operation name per Span.SetOperationName.
+// StartChildSpan is a simple helper to start a child span given only its
+// parent (per StartSpanOptions.Parent) and an operation name per
+// Span.SetOperationName.
 func StartChildSpan(parent Span, operationName string) Span {
-	return parent.Tracer().StartSpanWithOptions(StartSpanOptions{
-		OperationName: operationName,
-		Parent:        parent,
-	})
+	return parent.Tracer().StartSpan(
+		operationName,
+		Reference(RefBlockedParent, parent.SpanContext()),
```

Comment:
The comment for RefBlockedParent implies this is a change of semantics. StartChildSpan says nothing about blocking its parent.